### ISSUE TITLE
isKeyPressed right shift key always returns true after holding down both shift keys

### ISSUE THREAD

**Issue by so3arbelnox** [libgdx/libgdx#1570]
What steps will reproduce the problem?
1. Hold down both left and right shift key. Gdx.input.isKeyPressed returns true for both as expected.
2. Release keys. isKeyPressed returns true for right shift key still. Furthermore, returns true for right shift until it is pressed again.

Using libgdx 0.9.9

**Comment by FlukeFan** [comment]
Hi, I'm seeing the same behaviour using libgdx 1.5.2 on Windows 7, 64-bit using lwgjl backend.

As this has already been closed, do I need to create a new issue?  or (ideally) can this one be re-opened?

**Comment by NathanSweet** [comment]
What backend? If there is a problem it probably isn't in libgdx. Eg probably you need to file an issue with lwjgl.

**Comment by FlukeFan** [comment]
The backend is lwgjl.

If I get a chance, I'll try and determine if the bug is in lwgjl or libgdx.  Should this issue be re-opened in the meantime?

**Comment by FlukeFan** [comment]
Found this thread:  http://forum.lwjgl.org/index.php?topic=5210.msg27808#msg27808

**Comment by loogie** [comment]
Hello!

 

You have a new message, please read <http://www.mum10.com/producer.php?dz6d>

 

loogie@basicelementdesign.com

**Comment by wcko87** [comment]
Hello, I would like to request that this issue be reopened.
This problem continues to exist in libgdx version 1.9.9.

Here is a recording of the issue: (using libgdx 1.9.9, windows 10)
https://gfycat.com/ForsakenTerrificCaribou

I have read through the thread above. As far as I can tell, this issue is specific to libgdx (or possibly lwjgl). It does not happen on any other application on my PC (e.g. notepad, or the input display I am using in that video).

Both `isKeyPressed()` and `InputProcessor` share this issue. Also, I have found no way thus far in libgdx to identify whether LSHIFT has been released after releasing both LSHIFT and RSHIFT simultaneously. libgdx seems to truly believe LSHIFT continues to be held.

The code I was using to test this: https://pastebin.com/dX5BWTx2
